2.segmentation/scripts/0.segment_images.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+#
+
+# In[1]:
+
+
+import pathlib
+
+import cv2
+import numpy as np
+
+# from segment_anything import SamPredictor
```

Comment:
Recommend moving this code comment to where the segment_anything import happens.

Reply:
removed

Reply:
removed